extensions/wikia/DesignSystem/i18n/de.json

```diff
@@ -72,5 +72,22 @@
   "global-navigation-wikis-community-central": "Community Deutschland",
   "global-navigation-wikis-explore": "Wikis erforschen",
   "global-navigation-wikis-fandom-university": "Fandom Universität",
-  "global-navigation-wikis-header": "Wikis"
+  "global-navigation-wikis-header": "Wikis",
+  "notifications-no-notifications-message": "Noch keine Benachrichtigungen.",
+  "notifications-mark-all-as-read": "Alle als gelesen markieren",
+  "notifications-replied-by-multiple-users-with-title": "__mostRecentUser__ und __number__ weitere Benutzer haben auf __postTitle__ <b>geantwortet</b>",
```

Comment:
Oh my this is going to be painful. Will address this as I move on to the notification titles.